config/storage_backup_url_test.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import "testing"
+
+func TestGetStorageBackupURL(t *testing.T) {
+	StorageFallbackURLs = map[string]string{"https://storj.livepeer.com/catalyst-recordings-com/hls": "https://google.livepeer.com/catalyst-recordings-com/hls"}
```

Comment:
do we need to restore the default value after tests?
```suggestion
	StorageFallbackURLs = map[string]string{"https://storj.livepeer.com/catalyst-recordings-com/hls": "https://google.livepeer.com/catalyst-recordings-com/hls"}
	defer func() { StorageFallbackURLs = nil }()
```